### PR TITLE
Fix #1649, Add Functional Test for EVS Reset Filters API.

### LINKED ISSUE #1649: Add functional tests for cFE EVS Reset Event Filter APIs
**Is your feature request related to a problem? Please describe.**
Need open source functional tests for certifiability

**Describe the solution you'd like**
Add functional tests for cFE EVS Reset Event Filter APIs
– CFE_EVS_ResetFilter - Resets the calling application's event filter for a single event ID.
– CFE_EVS_ResetAllFilters - Resets all of the calling application's event filters.

**Describe alternatives you've considered**
N/A

**Additional context**
N/A

**Requester Info**
Alex Campbell GSFC

### PR DESCRIPTION
**Pull request by zanzaben**
**Describe the contribution**
Fixes #1649
Add Functional Test for EVS Reset Filters APIs.

**Testing performed**
Build and run unit test

**Expected behavior changes**
No impact to behavior

**System(s) tested on**
Ubuntu 20.04

**Contributor Info - All information REQUIRED for consideration of pull request**
Alex Campbell GSFC

### CODE REVIEW

File: modules/cfe_testcase/src/evs_filters_test.c [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+ * Includes
+ */
+
+#include "cfe_test.h"
+
+void TestResetFilters(void)
+{
+    UtPrintf("Testing: CFE_EVS_ResetFilter, CFE_EVS_ResetAllFilters");
+
+    UtAssert_INT32_EQ(CFE_EVS_ResetFilter(1), CFE_SUCCESS);
```
Comment (skliper) [inline modules/cfe_testcase/src/evs_filters_test.c:40]: Suggest a comment here, something like - `Test logic below depends on the test case registering an EID of 1 and not registering 0, and the resets in theory could impact test behavior/management related to events.  Although the expectation is either all or none of an EID would be filtered (no use case for a "counting" filter within the test app) so for nominally this is no impact.`  Provides clues for maintenance.

**Review by skliper — CHANGES_REQUESTED** [review]

**Comment by astrogeco** [comment]
[CCB:2021-08-04](https://github.com/nasa/cFS/wiki/Community-CCB-Agenda:-August-04-2021) APPROVED

**Comment by astrogeco** [comment]
@zanzaben can you fix the conflicts?

**Comment by zanzaben** [comment]
> @zanzaben can you fix the conflicts?

conflicts resolved.

**Comment by astrogeco** [comment]
Cool, how about @skliper's comment?

**Review by skliper — APPROVED** [review]

**Comment by astrogeco** [comment]
Resolved one more conflict and merged manually. Workflows run successfully, see https://github.com/astrogeco/cFE/runs/3292238330

**Pull request merged**